deliverables/README.md

```diff
@@ -12,16 +15,37 @@ Sign Up for Cloud9 and create a basic Rails project.
 ####Required
 
 What is an "interpreted" language?<br>
+In an interpreted language, the code we write is translated and run through an interpreter which performs the user's desired actions.
+
 What is IRB?<br>
+IRB stands for interactive Ruby. It interactively executes whatever Ruby code we write one line at a time.
+
 What are Objects?<br>
+Every thing in Ruby is an object. Objects are classified by the characteristics which make up a class of objects. In addition to an objects defining characteristics, you can perform functions/methods on objects.
+
 What are Methods?<br>
+Methods, or functions. are a set of instructions bound to an object and have access to its instance variables. You call methods on objects.
+
 What are Classes?<br>
+A class is the blueprint for which objects are created. Classes classify objects and each class of object has certain characteristics and functions associated to them.
+
 What are Blocks?<br>
+Blocks in Ruby are sets of instructions, placed inside curly braces, which can take arguments and return values and are invoked using a yield statement. They are syntax literals for Proc objects.
+
 What is an Array?<br>
+An array is a list of stored information. The list in an array has order. 
+
 What is an Iterator?<br>
+An iterator specifies the way a method might go through a collection of objects. Examples of iterators are "each" which returns each value of a collection one by one and "collect" which returns the entire collection of objects.
+
 What are hashes?<br>
+A hash, also known as a dictionary, stores pairs of related information. Each data set has a reusable lable to call on it: a key and a value, so as opposed to an array, order in a hash doesn't matter which is why it's like a dictionary that has a vocab word (key) and a definition (value).
+
```

Comment:
Also order doesn't matter in Arrays and Also what is important to understand about Hashs is that each KEY must be unique

This is NOT valid:

    {a: 1, b:2, a:3}


This IS valid:

    {x: 1, xy:2, xyz:3}